assets/src/components/list-item/style.scss

```diff
@@ -0,0 +1,16 @@
+.list-items {
+
+	&--list-style-disc {
```

Comment:
Let's not nest selectors/use SCSS placeholders here. It's not regarded as a best practice since searching the codebase for a specific selector can become pretty hard.